notebooks/model_monitoring/solutions/model_monitoring_vertex.ipynb

```diff
@@ -0,0 +1,1396 @@
+{
```

Comment:
<p>Maybe explain what the Jensen-Shannon divergence and how it is used in this context. Same thing with "High Linfty". Information regarding when the metric values can be considered as significant or not may be also helpful. Right now, it's not super what to do with this output without a little more guidance. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>